certifi/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .core import where
+from .core import what, where
```

Comment:
This is broken

Reply:
Fixed.

Reply:
This doesn't seem resolved from what GitHub is showing me

Reply:
Fixed.